doc/install/installers_platform_selector.html

```diff
@@ -69,7 +69,7 @@ <h5 class="m-0 font-weight-bold">
     <div id="collapseWindows" class="collapse" aria-labelledby="windows" data-parent="#downloadAccordion">
       <div class="card-body">
         <a class="btn btn-primary font-weight-bold shadow-sm mt-3 mb-3" role="button"
-          href="https://github.com/mne-tools/mne-installers/releases/download/v1.0.0/MNE-Python-1.0.0_0-Windows.exe">
+          href="https://github.com/mne-tools/mne-installers/releases/download/v1.0.0-post1/MNE-Python-1.0.0_1-Windows.exe">
```

Comment:
https://github.com/mne-tools/mne-installers/releases/download/v1.0.0-post1/MNE-Python-1.0.0_1-Windows.exe is dead

Reply:
... ahh, release not complete yet. Let me finish testing Windows now!